bindata/network/ovn-kubernetes/006-ovs-node.yaml

```diff
@@ -47,6 +47,14 @@ spec:
             source "/env/${K8S_NODE}"
             set +o allexport
           fi
+          export SYSTEMD_IGNORE_CHROOT=yes
+          # Check to see if ovs is provided by the node:
+          if systemctl status ovsdb-server; then
+            echo "openvswitch is running in systemd"
+            # Don't need to worry about restoring flows; this can only change if we've rebooted
+            exec tail -F /var/log/openvswitch-host/ovs-vswitchd.log /var/log/openvswitch-host/ovsdb-server.log
+          fi
```

Comment:
Do you mean /var/log/openvswitch  instead of openvswitch-host? 

Is the goal to block here, and never return?

Reply:
No, this is correct. /var/log/openvswitch is not a host directory.
The goal is to block here. We need to keep a process running so kubelet doesn't think the pod has died.

Reply:
> No, this is correct. /var/log/openvswitch is not a host directory.

Then shouldn't e.g. 

```
tail -F --pid=$(cat /var/run/openvswitch/ovs-vswitchd.pid) /var/log/openvswitch/ovs-vswitchd.log &
```

below also use ```/var/log/openvswitch-host``` ?

Reply:
no... because that's for the containerized case.